.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+      # Push events on main branch
+      - master
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - v0.*
```

Comment:
what happens when we release 1.? I think we can just do something like v*.* or just v*